android/src/main/kotlin/com/aheaditec/freerasp/ScreenProtector.kt

```diff
@@ -0,0 +1,130 @@
+package com.aheaditec.freerasp
+
+import android.annotation.SuppressLint
+import android.app.Activity
+import android.app.Activity.ScreenCaptureCallback
+import android.content.Context
+import android.content.pm.PackageManager
+import android.os.Build
+import android.util.Log
+import android.view.WindowManager.SCREEN_RECORDING_STATE_VISIBLE
+import androidx.annotation.RequiresApi
+import androidx.core.content.ContextCompat
+import androidx.lifecycle.DefaultLifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+import com.aheaditec.talsec_security.security.api.Talsec
+import java.util.function.Consumer
+
+@SuppressLint("StaticFieldLeak")
```

Comment:
Not liking activity in a static object. Can we make ScreenProtector a class and instantiate it in MethodCallHandler? enable() could be call in the start method of MethodCallHandler. I know it is not very nice solution either, but we would at least keep it consistent with businessRASP